packages/common/src/types/rollup.ts

```diff
@@ -17,18 +17,21 @@ export enum ERollUpState {
   Failed = 'Failed',
 }
 
-export type TRollUpHistory = TDbRecord<{
+export type TRollUpHistory = {
   databaseName: string;
   currentMerkleTreeRoot: string;
   previousMerkleTreeRoot: string;
   // Previous name `txId` is changed to `transactionObjectId`,
   // txId is not a good name it's alias of tx hash
   transactionObjectId: ObjectId;
   proofObjectId: ObjectId;
-  transactionHash: string;
-  status: ETransactionStatus;
-  error: string;
-}>;
+};
+
+export type TRollUpHistoryRecord = TDbRecord<TRollUpHistory>;
+
+// Compound Type
+export type TRollUpTransactionHistory = TRollUpHistory &
+  Omit<TDbRecord<TTransaction>, '_id'>;
```

Comment:
It supposed to be `WithoutId<TTransactionRecord>`

Reply:
done